FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDemoTests.m

```diff
@@ -63,6 +63,28 @@ - (void)testViewSnapshotWithUIAppearance
   FBSnapshotVerifyView(control, nil);
 }
 
+- (void)testViewSnapshotWithUIAppearanceResizing
+{
+  // If this works properly you should be able to read the whole "Click me!" title in the snapshot
```

Comment:
No need for the comment here IMO - it's obvious from the snapshots themselves

Reply:
You are right, removed :)